src/wordlist.c

```diff
@@ -387,7 +387,9 @@ void wordlist_hybrid_fix_state(void)
 
 static double get_progress(void)
 {
-	int64_t pos, size;
+	struct stat file_stat;
+	long pos;
```

Comment:
This is a bug: `pos` must be `int64_t` or similar in jumbo. It's only `long` in core.